test/library/standard/Path/victor-ludorum/joinPath/testcases.chpl

```diff
@@ -0,0 +1,30 @@
+use Path;
+
+// writing tests for path joining where merging path name starts with ('/')
+writeln(joinPaths("/foo/bar", "/baz"));
```

Comment:
It looks like you missed updating the name here.  Did you try running this through start_test to ensure it worked?

Reply:
Oh sorry Mam I will update it .